natgeo/mortar
# [SYTO-731] Carrying over some css properties to new class names

Carrying over some css properties to new class names. updated doc page to use new class names in the existing examples on the documentation site

https://jira.natgeo.com/browse/SYTO-731

To test:
Pull bugfixes/SYTO-731
Run `gulp serve`
Check heading section of http://192.168.1.129.xip.io:3000/typography/ against AC on https://jira.natgeo.com/browse/SYTO-731
